### PR TITLE
fixes #3250: cron tasks in rpm install of foreman use foreman-rake

### PR DESCRIPTION
**Pull request by dmitri-d**

### CODE REVIEW

File: extras/packaging/rpm/sources/foreman.cron.d [hunk 1]
```diff
@@ -5,16 +5,16 @@ RAILS_ENV=production
 FOREMAN_HOME=/usr/share/foreman
 
 # Clean up the session entries in the database
-15 23 * * *     foreman    cd ${FOREMAN_HOME} && /usr/bin/ruby193-rake db:sessions:clear >/var/log/foreman/cron.log 2>&1
+15 23 * * *     foreman    cd ${FOREMAN_HOME} && /usr/bin/foreman-rake db:sessions:clear >/var/log/foreman/cron.log 2>&1
```
Comment (domcleal) [inline extras/packaging/rpm/sources/foreman.cron.d]: The "cd" is no longer required and foreman-rake's in /usr/sbin, not bin/.

**Comment by dmitri-d** [comment]
@domcleal: how about now?

**Comment by domcleal** [comment]
@witlessbird looks better, thanks.  Did you test it? :)

**Comment by dmitri-d** [comment]
@domcleal: not yet, waiting for the scratch build to finish.

**Comment by dmitri-d** [comment]
@domcleal: works on f19

**Comment by domcleal** [comment]
Thanks @witlessbird, merged as b5ff020.

**Pull request closed without merging**